WordPress/twentytwentyfive
# Add font sizes

**Description**
A first go at adding the font sizes.
Closes https://github.com/WordPress/twentytwentyfive/issues/36

**Screenshot**
![font-size-control-tt5](https://github.com/user-attachments/assets/09341bc5-9a02-4d6b-9cb0-8ce471df95e2)


**Testing Instructions**
1. Apply the PR
2. Confirm that the sizes are applied correctly to the body, headings, and the updated blocks, for example
Site title, tagline, search, code, post date.
